Makefile

```diff
@@ -177,8 +177,10 @@ prepare-minor:
 prepare-major:
 	./hack/prepare-release.sh major
 
+release-notes:
```

Comment:
Who calls this?

Reply:
This target is used by users to test release notes with something like

```bash
make release-notes WHAT="v0.65.1 v0.65.2"
```

This way to can check how they look before doing a release and if something is missing change the PR description is not ok.

Reply:
Understood